src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1120,33 +1289,104 @@ CHIP_ERROR CASESession::SetEffectiveTime(void)
     return ASN1ToChipEpochTime(effectiveTime, mValidContext.mEffectiveTime);
 }
 
-CHIP_ERROR CASESession::HandleErrorMsg(const System::PacketBufferHandle & msg)
+void CASESession::OnSuccessStatusReport()
 {
-    ReturnErrorCodeIf(msg->Start() == nullptr || msg->DataLength() != sizeof(SigmaErrorMsg), CHIP_ERROR_MESSAGE_INCOMPLETE);
+    ChipLogProgress(SecureChannel, "Success status report received. Session was established");
```

Comment:
Shouldn't this check state to make sure that we actually proceeded through all the steps properly?